db/migrate/20180829195404_add_omniauth_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddOmniauthToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :provider, :string
+    add_column :users, :uid, :string
```

Comment:
I don't see my previous comment - I had suggested that this be `urs_uid` if there's a chance of other user ids being incorporated.